store/modules/data/mutations.js

```diff
@@ -1,5 +1,5 @@
 // adapted from https://stackoverflow.com/questions/53446792/nuxt-vuex-how-do-i-break-down-a-vuex-module-into-separate-files
-
+const MAX_NUM_DATAPOINTS_FOR_MEAN = 5;
```

Comment:
This value should be defined in only one file and imported into any files and tests that need to use it